src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java

```diff
@@ -201,12 +201,12 @@ public void fromXML(final XMLDocument xmlDoc) {
             voice = xmlDoc.getNodeValue(CON_VOICE_EXPR);
             String voiceXStr = xmlDoc.getNodeValue(CON_VOICEX_EXPR);
             if (voiceXStr != null && voiceXStr.length() > 0) {
-                voiceX = Integer.parseInt(voiceXStr);
+                voiceX = Long.parseLong(voiceXStr);
```

Comment:
Related with my previous comment - we won't have to `parseLong`.

Reply:
Review comment has been addressed.